### PR TITLE
add challenges

### PR DESCRIPTION
**Pull request by Antonio-IGG**

### CODE REVIEW

File: staff/antonio-izquierdo/challenges/calculattor-app/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Calculator App
+
+Esta aplicación es una calculadora básica. Permite realizar las siguientes operaciones matemáticas: sumar, restar, multiplicar y dividir.
```
Comment (betoDavila86) [inline staff/antonio-izquierdo/challenges/calculattor-app/README.md]: I cannot understand Spanish. Would you mind it translating to English? Thanks mate!

File: staff/antonio-izquierdo/challenges/calculattor-app/calculate.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+var current = ''
+var aux = ''
+var resetValue = '';
+
+//SELECCIONAMOS TODOS LOS BOTONES. DESPUÉS SELECCIONAMOS EL DISPLAY.
+var buttons = document.querySelectorAll('button');
+var display = document.querySelector('span');
+
+//"aux" recoge el valor y lo añade a "current", después vaciamos el display con una empty string [''];
+aux = current
```
Comment (betoDavila86) [inline staff/antonio-izquierdo/challenges/calculattor-app/calculate.js]: Semicolons at the end of the declarations or statements.

**Review by betoDavila86 — CHANGES_REQUESTED** [review]

File: staff/antonio-izquierdo/challenges/calculattor-app/README.md (hunk 1 above)
Comment (Antonio-IGG) [reply staff/antonio-izquierdo/challenges/calculattor-app/README.md]: Ok! Thank U

**Review by simonbesteiro — APPROVED** [review]
Review done

**Pull request merged**